tests/regression/test_exodus_mesh.py

```diff
@@ -0,0 +1,23 @@
+import pytest
```

Comment:
Add `from __future__ import absolute_import, print_function, division` to satisfy the flake8 requirements